rotors_gazebo/resource/neo11.yaml

```diff
@@ -0,0 +1,14 @@
+# NEO11 Hex vehicle parameters
+mass: 3.42
+inertia: {xx: 0.0608 , xy: 0.0, xz: 0.0, yy: 0.0688, yz: 0.0, zz: 0.1489}
+
+# angle should be expressed in rad
+# direction 1 means counter clockwise, -1 clockwise
+# The order of the rotors matters for the calculation of the allocation matrix.
+rotor_configuration:
+  '0': {angle: 0.52359877559, arm_length: 0.2895, rotor_force_constant: 1.269e-05, rotor_moment_constant: 1.6754e-2, direction: 1.0}
```

Comment:
The torque is given by rotor_moment_constant* rotor_force_constant*omega^2. This close enough to firefly (1.6e-2). Why do you think it is too high?